### PR TITLE
Learn tag

### PR DESCRIPTION
**Pull request by cpinzon**
This feature branch adds support for a learn tag. 

The learn tag is supposed to notify the developer of learning resources available for the specific API that they are viewing. It would be added in the following manner: 

/**
- @namespace The Utils object provides useful utility functions and properties.  
- @toc {Utilities} Utils
- @featureID blackberry.utils
- @learns {W3C Schools} http://www.w3schools.com/ This link redirects to W3C Schools, where they offer free tutorials in all web development technologies.
  */

I will send a picture of what the addition of the tag to an API would look like, as I seem unable to figure how to add a picture to this discussion.

### CODE REVIEW

**Comment by tneil** [comment]
Hi Carolina... does the feature allow for multiple @learns tags?

File: build/BBTemplate/class.tmpl [hunk 1]
```diff
@@ -93,6 +111,14 @@
 		<script src="server.js"></script>
 	</head>
 	
+
```
Comment (jeffheifetz) [inline build/BBTemplate/class.tmpl:114]: I think the description should go below the beta tag

**Comment by cpinzon** [comment]
Tim, I've added a few changes to ensure multiple tags are supported.

**Comment by jeffheifetz** [comment]
Does anyone else think the learn tags should go at the bottom?

**Comment by cdelcol** [comment]
let's just keep them where they are for now. They can always be moved later.

**Pull request merged**